Essentials/src/config.yml

```
@@ -758,6 +758,16 @@ protect:
     # Monsters won't follow players.
     # permission essentials.protect.entitytarget.bypass disables this.
     entitytarget: false
+    # Prevent certain transformations.
+    transformation:
+      creeper-charge: false
+      villager-infection: false
+      villager-cure: false
+      villager-to-witch: false
+      pig-transformation: false
+      zombie-drowning: false
+      # Prevent brown and red mooshrooms switch on lightning strike
+      mooshroom-switching: false
```

Each option should be documented since the names aren't always clear:

```suggestion
    transformation:
      # Prevent creepers becoming charged when struck by lightning.
      creeper-charge: false
      # Prevent villagers becoming zombie villagers.
      villager-infection: false
      # Prevent zombie villagers being cured.
      villager-cure: false
      # Prevent villagers becoming witches when struck by lightning.
      villager-to-witch: false
      # Prevent pigs becoming zombie pigmen when struck by lightning.
      pig-transformation: false
      # Prevent zombies turning into drowneds.
      zombie-drowning: false
      # Prevent mooshrooms changing colour when struck by lightning.
      mooshroom-switching: false
```